### PR TITLE
Make WorkingDayService Constructor Internal

### PR DESCRIPTION
**Pull request by ghost**
Made the `WorkingDayService` constructor internal, forcing use of the `WorkingDayServiceBuilder`, and updated the tests accordingly.

Also added an extra `WorkingDayServiceBuilder` extension to use `params` for non-working days in `DaysOfTheWeekNonWorkingDaySource`.

### CODE REVIEW

**Review by lparkermg — DISMISSED** [review]
LGTM

File: src/Tools.WorkingDayService.Tests/WorkingDayServiceAddWorkingDayTests.cs [hunk 1]
```diff
@@ -2,10 +2,11 @@
 // Copyright (c) Cognisant. All rights reserved.
 // </copyright>
 
+
```
Comment (jbradford) [inline src/Tools.WorkingDayService.Tests/WorkingDayServiceAddWorkingDayTests.cs]: Whitespace (surprised it was allowed by style cop!)

File: src/Tools.WorkingDayService.Tests/WorkingDayServiceIsNonWorkingDayTests.cs [hunk 2]
```diff
@@ -14,31 +14,25 @@ namespace CR.Tools.WorkingDayService.Tests
...
-        /// </summary>
-        [Test]
-        public static void WorkingDayServiceWithNoSourcesReturnsNoDayAsNonWorkingDay() => Assert.IsFalse(new WorkingDayService(new List<NonWorkingDaySource>()).IsNonWorkingDay(DateTime.Now));
-
         /// <summary>
         /// Test to check that a Working Day Service with one Source considers a Working Day a Working Day.
         /// </summary>
         [Test]
-        public static void WorkingDayServiceWithOneSourceReturnsFalseWhenIsWorkingDayIsNonWorkingDayCalledOnAWorkingDay() => Assert.IsTrue(new WorkingDayService(new List<NonWorkingDaySource> { new MondayNonWorkingDayTestSource() }).IsWorkingDay(new DateTime(2018, 5, 15)));
+        public static void WorkingDayServiceWithOneSourceReturnsFalseWhenIsWorkingDayIsNonWorkingDayCalledOnAWorkingDay() => Assert.IsTrue(((WorkingDayService)WorkingDayServiceBuilder.New().AddDaysOfTheWeekNonWorkingDaySource(DayOfWeek.Monday)).IsWorkingDay(new DateTime(2018, 5, 15)));
```
Comment (jbradford) [inline src/Tools.WorkingDayService.Tests/WorkingDayServiceIsNonWorkingDayTests.cs]: Might be worth putting the implementation on a second line for readability.

**Review by jbradford — DISMISSED** [review]
Code looks ok. A few very long lines that are quite hard to read.

**Review by rcknight — APPROVED** [review]

**Pull request merged**